geo/src/algorithm/simplify.rs

```diff
@@ -192,7 +198,13 @@ pub trait Simplify<T, Epsilon = T> {
 /// This operation uses the [Ramer–Douglas–Peucker algorithm](https://en.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm)
 /// and does not guarantee that the returned geometry is valid.
 ///
-/// An epsilon less than or equal to zero will return an unaltered version of the geometry.
```

Comment:
A related question is can we alter our algorithm such that `epsilon == 0` will remove collinear points, but nothing else?

Reply:
Maybe with distance and area calculations it's not feasible to have epsilon == 0 from a robustness perspective...

Reply:
Hmm I don't know that either algorithm can be made to work precisely like that. I can have a look at "the literature".